<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1506</width>
    <height>926</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(85, 0, 0);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="widget_2" native="true">
    <property name="geometry">
     <rect>
      <x>870</x>
      <y>240</y>
      <width>311</width>
      <height>411</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgba(255,255,255,20);
border-top-right-radius: 50px;
border-bottom-right-radius: 50px;
</string>
    </property>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>291</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 700 14pt &quot;Segoe UI&quot;;
color: rgb(255, 255, 255);
background-color: rgba(255,255,255,0);
</string>
     </property>
     <property name="text">
      <string>Username</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>120</y>
       <width>291</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 700 14pt &quot;Segoe UI&quot;;
color: rgb(255, 255, 255);
background-color: rgba(255,255,255,0);
</string>
     </property>
     <property name="text">
      <string>Password</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="username_lineedit">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>70</y>
       <width>291</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgba(0,0,0,0);
border: none;
border-bottom: 1px solid rgba(255,0,0,200);
color: rgba(0,10,0,240);
padding-bottom: -15px;
color: rgb(255, 255, 255);
font: 700 11pt &quot;Segoe UI&quot;;</string>
     </property>
     <property name="placeholderText">
      <string>Username</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="password_lineedit">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>160</y>
       <width>291</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgba(0,0,0,0);
border: none;
border-bottom: 1px solid rgba(255,0,0,200);
color: rgba(0,10,0,240);
padding-bottom: -15px;
color: rgb(255, 255, 255);
font: 700 11pt &quot;Segoe UI&quot;;</string>
     </property>
     <property name="echoMode">
      <enum>QLineEdit::Password</enum>
     </property>
     <property name="placeholderText">
      <string>Password</string>
     </property>
    </widget>
    <widget class="QPushButton" name="login_button">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>220</y>
       <width>291</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgba(0,0,0,0);
border: none;
color: rgb(255, 255, 255);
font: 700 12pt &quot;Segoe UI&quot;;
border-bottom: 1px solid rgba(255,0,0,200);
border-top: 1px solid rgba(255,0,0,200);
border-left: 1px solid rgba(255,0,0,200);
border-right: 1px solid rgba(255,0,0,200);</string>
     </property>
     <property name="text">
      <string>login</string>
     </property>
    </widget>
    <widget class="QLabel" name="forgot_password_label">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>280</y>
       <width>291</width>
       <height>16</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true"> font: 9pt &quot;Segoe UI&quot;;
 color: rgb(250, 144, 169);
background-color: rgba(255,255,255,0);</string>
     </property>
     <property name="text">
      <string>forgot your password</string>
     </property>
    </widget>
    <widget class="QLabel" name="signin_label">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>300</y>
       <width>291</width>
       <height>16</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true"> font: 9pt &quot;Segoe UI&quot;;
 color: rgb(250, 144, 169);
background-color: rgba(255,255,255,0);</string>
     </property>
     <property name="text">
      <string>sign in</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="widget" native="true">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>240</y>
      <width>311</width>
      <height>411</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgba(0,0,0,80);
background-repeat: no-repeat;
background-position: up;
border-top-left-radius: 50px;
border-bottom-left-radius: 50px;
background-image: url(:/iut/photo19216140080.jpg);</string>
    </property>
   </widget>
   <widget class="QWidget" name="widget_3" native="true">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>240</y>
      <width>311</width>
      <height>411</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgba(0,0,0,80);
background-repeat: no-repeat;
background-position: up;
border-top-left-radius: 50px;
border-bottom-left-radius: 50px;</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1506</width>
     <height>17</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
